jetscape_analysis/analysis/reader/parse_ascii.py

```diff
@@ -270,6 +347,7 @@ def _parse_header_line_format_v2(line: str) -> HeaderInfo:
 # Register header parsing functions
 _file_format_version_to_header_parser = {
     2: _parse_header_line_format_v2,
+    3: _parse_header_line_format_v3,
```

Comment:
Can you please try to add some tests? If I recall correctly, they're rudimentary, but they're a bit useful. See them here: `JETSCAPE-analysis/tree/master/tests)/test_parsing.py` (there are some instructions on how they're generated in the comments)

Reply:
yes, I can take a look at adding the tests